internal/db/mongo/db.go

```diff
@@ -1,34 +1,30 @@
+// Package mongo implements mongo db interactions functionality.
 package mongo
 
 import (
-	"fmt"
 	"time"
 
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 	mgo "gopkg.in/mgo.v2"
 	"gopkg.in/mgo.v2/bson"
 
-	"github.com/oleg-balunenko/logs-converter/model"
+	"github.com/oleg-balunenko/logs-converter/internal/models"
 )
 
-const prefix = "db/mongo"
-
-// DB stores mongo db connection details
-type DB struct {
-	Session    *mgo.Session
+// db stores mongo db connection details
+type db struct {
+	session    *mgo.Session
 	database   *mgo.Database
 	collection *mgo.Collection
 }
 
 // NewMongoDBConnection establishes connection with mongoDB and return DBName object
-func NewMongoDBConnection(url, dbName, collectionName, username, password string) *DB {
-
+func NewMongoDBConnection(url, dbName, collectionName, username, password string) (*db, error) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported func NewMongoDBConnection returns unexported type *mongo.db, which can be annoying to use](https://app.codacy.com/app/oleg.balunenko/logs-converter/pullRequest?prid=4555937)